test/fixtures/users.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
Do you need this file?

Reply:
This file comes predefined with the project. I think we could use it later.